src/plugins/shared-variables/slate/variable-chip.tsx

```diff
@@ -0,0 +1,33 @@
+import { VariableType } from "@concord-consortium/diagram-view";
+import { observer } from "mobx-react";
+import React from "react";
+
+interface IProps {
+  variable: VariableType;
+}
+
+export const VariableChip: React.FC<IProps> = observer(({variable}) => {
+  const name = variable.name;
+  const value = variable.computedValue;
+  const valueString = variable.computedValueWithSignificantDigits;
+  const unit = variable.computedUnit || "";
+  const showValue = value !== undefined && !isNaN(value);
```

Comment:
I'm torn on this. The type of computedValue doesn't allow `null`, so it seems weird to refer to `null` in the condition when it should never hapen. I suppose a very strict typescript compiler might complain that the null check is unnecessary. But I've also read how it has been bad practice to refer directly to `undefined` in code because of the legacy of being able to re-define it to some other value.  I haven't checked, but I'd guess that our build chain takes care of this in some way though. 

I think I'll leave it for now.